### PR TITLE
[GH-#210] Represent hashes as Bech32 with a prefix and signatures as Base64

### PR DESCRIPTION
**Pull request by d-velev**
Address serialization to be done in the wallet

### CODE REVIEW

**Comment by thepiwo** [comment]
@nedelcho-delchev-tues @cheezus1 @meivantodorov @gspasov can you review?

File: apps/aecore/lib/aecore/peers/worker.ex [hunk 1]
```diff
@@ -62,7 +62,7 @@ defmodule Aecore.Peers.Worker do
   def genesis_block_header_hash() do
     Block.genesis_block().header
     |> BlockValidation.block_header_hash()
-    |> Base.encode16()
+    |> Header.bech32_encode()
```
Comment (cytadela8) [inline apps/aecore/lib/aecore/peers/worker.ex]: For internal use I would leave binary representation.

File: apps/aecore/lib/aecore/chain/worker.ex [hunk 2]
```diff
@@ -58,7 +60,7 @@ defmodule Aecore.Chain.Worker do
 
   @spec get_block_by_hex_hash(String.t()) :: Block.t()
   def get_block_by_hex_hash(hash) do
```
Comment (cytadela8) [inline apps/aecore/lib/aecore/chain/worker.ex]: We should also rename functions and variable names with "hex" in them if they are no longer hex. You can potentially remove some of such functions and variables. I would suggest using binary representation internally and bech for UI (including API and logs).

**Comment by thepiwo** [comment]
@cheezus1 @gspasov can you please review?

File: apps/aecore/lib/aecore/peers/worker.ex (hunk 1 above)
Comment (d-velev) [reply apps/aecore/lib/aecore/peers/worker.ex]: We use it to compare the genesis hash we get from client info, I think this is fine.

Comment (thepiwo) [reply apps/aecore/lib/aecore/peers/worker.ex]: better to compare using binary, as we use binary representation internally

**Comment by d-velev** [comment]
As I said, serialization of addresses will be done in the wallet.

**Comment by thepiwo** [comment]
@d-velev can you resolve conflicts and merge?

**Comment by thepiwo** [comment]
I think you can merge the PR as well

**Pull request merged**